docs/connectivity/Agent.mdx

```diff
@@ -0,0 +1,290 @@
+
+An instrument outputs files to a directory each time it completes a run, which serve as inputs for a Ganymede flow. 
+
+- **Cron**: Run Action on a pre-set schedule
+
+<div class="text--center">
+  <img alt="Agent configuration - Cron" 
+      src="https://ganymede-bio.mo.cloudinary.net/agent/Agent_Cron_Config20230605.png"
+      width="500" />
+</div>
+&nbsp;
+
+#### Example use case for Cron mode
+
+A local script (the Normalizer) must be run once a day to poll and capture updates from telemetry devices for further processing in a Ganymede flow.
+
+- **Cloud Watcher**: Monitor specified directory in Ganymede Cloud Storage for new files, and execute upon observing new files matching specified pattern
+
+#### Example use case for Cloud Watcher mode
+
+Output data from a Ganymede flow is retrieved and cross-checked against instrument data to verify that the lab machine is producing expected results.
+
+- **Manual**: Upon Agent launch, the Agent code executes once and then exits.
+
+### Build Config: Action
+
+Actions are executed after the Listener and Normalizer (user-defined Python) have completed.
+
+- **Upload File**: Uploads file(s) to Ganymede Cloud Storage in the agent directory.  This data can be found in the File Explorer under "/_agents/<agent id\>/"
+
+#### Example use case for Upload File Action
+
+Files output by an instrument are stored in Ganymede as a source of truth.
+
+- **Run Flow**: Run a flow based on the input provided and as mediated by the flow input requirements. 
+
+#### Example use case for Run Flow Action
+
+An instrument outputs files to a directory as it completes runs, which are processed in Ganymede Cloud. 
+
+- **No-Op**: Complete Agent execution without performing an Action
+
+## Installing Agents
+
+To install an Agent, open the Ganymede application in a browser window and navigate to the Connection tab in the left sidebar.  Select the desired Agent by name and download the relevant Windows/Linux installation file.
+
+:::note
+
+Both Linux and Windows versions of the Agent are built using x86_64 architecture.  The Linux executable is Ubuntu-based and the Windows executable is Windows Server 2022.
+
+:::
+
+### Windows Installation
+
+After downloading the Agent, launch the installation file to complete Agent configuration
+
+<div class="text--center">
+  <img alt="Windows Agent installation"
+      src="https://ganymede-bio.mo.cloudinary.net/agent/Agent_Windows_Setup20230605.png"
+      width="500" />
+</div>
+&nbsp;
+
+- **Connection name** is the name seen by users in the instrument computer and in the Ganymede UI
+- **Variable definitions** are strings in the format "var_name=var_value", which allows users to set context variables to be used in the Normalizer
+- **Label** is a string that can be used to identify and group connections within Ganymede.  The label is visible in the Connections UI
+
+TODO: clarify how to specify variable definitions and label with examples
```

Comment:
@eriis1 @joshasharpe is there documentation on setting variables/labels for the windows agent somewhere?